make/compiler/Makefile.gnu

```diff
@@ -118,6 +118,21 @@ ifeq ($(shell test $(GNU_GPP_MAJOR_VERSION) -lt 4; echo "$$?"),1)
 WARN_GEN_CFLAGS += -Wno-pointer-sign
 endif
 
+#
+# 2016/03/28: Help to protect the Chapel compiler from a partially
+# characterized GCC optimizer regression when the compiler is being
+# compiled with gcc 5.X.  Issue will be pursued after the release
+#
+# Note that 0 means "SUCCESS" rather than "false".
+ifeq ($(shell test $(GNU_GPP_MAJOR_VERSION) -eq 5; echo "$$?"),0)
```

Comment:
I would do greater than or equal to instead of equal to. i.e. `-eq` => `-ge` especially since gcc will now be bumping major versions much more frequently: https://gcc.gnu.org/develop.html